### PR TITLE
add Peak District to the list. solving #3

### LINKED ISSUE #3: Provide some of the mountains and lakes in England in the `england.md` file
Let's add some based on Google and Wikipedia

### PR DESCRIPTION
**Pull request by flicksolutions**
This PR solves #3 in adding contents to the list.

### CODE REVIEW

**Comment by 0066cc** [comment]
Needs more mountains.

**Comment by dimpurr** [comment]
The mountain(s) already existed LGTM but may need a little more contents

**Review by 0066cc — CHANGES_REQUESTED** [review]
Needs more mountains. @flicksolutions

File: england.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Mountains and Lakes in England
 
 England is not very mountainous. However, it has some tall hills, and maybe a mountain or two depending on your definition.
+
+- Peak District National Park
```
Comment (0066cc) [inline england.md]: This is not a mountain.

Comment (flicksolutions) [reply england.md]: oh.. you're right... On it...

Comment (0066cc) [reply england.md]: For reference: ⛰️

**Review by 0066cc — APPROVED** [review]
Nice!

**Comment by dimpurr** [comment]
LGTM

**Comment by flicksolutions** [comment]
@0066cc I comleted the list. can you pls look at it again?

**Pull request merged**